messaging-activemq/src/main/java/org/wildfly/extension/messaging/activemq/DiscoveryGroupAdd.java

```diff
@@ -153,7 +152,7 @@ static DiscoveryGroupConfiguration createDiscoveryGroupConfiguration(final Strin
         final long refreshTimeout = config.getRefreshTimeout();
         final long initialWaitTimeout = config.getDiscoveryInitialWaitTimeout();
 
-        final BroadcastEndpointFactory endpointFactory = new ChannelBroadcastEndpointFactory(channel, channelName);
+        final BroadcastEndpointFactory endpointFactory = new ForkChannelBroadcastEndpointFactory(channel, channelName);
```

Comment:
While the channel is question is a ForkChannel by default, it is not necessarily so.  Thus the name of this class isn't quite appropriate.